demo/lib/distro.bash

```diff
@@ -119,8 +119,8 @@ ubuntu-20_04-image-url() {
     echo "https://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-amd64.img"
 }
 
-ubuntu-21_04-image-url() {
-    echo "https://cloud-images.ubuntu.com/releases/hirsute/release/ubuntu-21.04-server-cloudimg-amd64.img"
+ubuntu-22_04-image-url() {
```

Comment:
This would add a duplicate `ubuntu-22_04_image-url()` function. Removing `ubuntu-21_04-image-url()` will be enough.

Reply:
Indeed, I totally missed the fact that we supported 22.04 already. I changed also the commit message to reflect the fact that we are actually removing 21.04 support.